### DISCUSSION TITLE
[Custom themes] Let's make it possible to set the Terrastories color scheme for a community in Theme

### DISCUSSION THREAD

**Discussion by rudokemper** [Terrastories/terrastories#755] [Temporarily Archived Tickets]
## Context:
Now that we have a Theme model in Rails, let's make it possible to set the core Terrastories colors there. We can create three fields - primary color, secondary color, tertiary color - that ask for a hex code. Then, we can apply those colors to all of the areas where we are currently using branded colors. (For example, the orange, green and dark blue in the screenshow below.)

## Acceptance criteria:
[] Admin user logs into Terrastories and accesses the Theme menu.
[] Add hex codes for primary and secondary color fields.
[] These colors are now applied to all places where we are using the standard Terrastories orange (`$orange`), green (`$teal`) and dark blue (`$blue`).

## Assets
Please link or attach any required assets, mocks, or copy
![image](https://user-images.githubusercontent.com/31662219/103093787-5ca51d80-45c9-11eb-9bf0-c4ffbec2bbf5.png)

**Comment by everly-gif** [comment]
I would like to work on this if its still up for grabs !

**Comment by rudokemper** [comment]
@everly-gif You got it :)
